src/pages/BusinessSpecifics/BusinessSpecifics.tsx

```diff
@@ -2,14 +2,35 @@ import { FormControl, TextField, Button, MenuItem, Select } from "@mui/material"
 import { useState } from "react"
 import { currencies } from "./currencies"
 
-export const BusinessSpecifics = () => {
 
-    const [BusinessName, setBusinessName] = useState("")
-    const [LocationName, setLocationName] = useState("")
+export const BusinessSpecifics = () => {
+    const url = 'https://invent0ry-back3nd.herokuapp.com/businesses/'
+    const [businessName, setBusinessName] = useState("")
+    const [locationName, setLocationName] = useState("")
     const [currency, setCurrency] = useState(currencies.USD)
 
     const currencyList = Object.keys(currencies).map(key => (<MenuItem value={key}>{key}</MenuItem>))
 
+    function handleSubmit(){

```

Comment:
this handle submit has a "event" argument
`function handleSubmit(e:React.SyntheticEvent){
// to prevent the form to be submitted 
e.preventDefault()

}`